### ISSUE TITLE
How to view 'debug' output?

### ISSUE THREAD

**Issue by NorthboundTrain** [davidnussio/vscode-jq-playground#71]
Running this through `jqpg` to both the console and the editor does not show the debug output:

```
jq '.[] | debug | .bar'
{ "foo": { "bar": true, "baz": false } }

```

My hope was that this would show up in some way similar to `jq` itself or up on jqplay.org:

```
$ echo '{ "foo": { "bar": true, "baz": false } }' | jq '.[] | debug | .bar'
["DEBUG:",{"bar":true,"baz":false}]
true
```

jqplay:
```
["DEBUG:",{"bar":true,"baz":false}]
true
```

Do I have to enable STDERR in the console, somehow? Thanks!

**Comment by davidnussio** [comment]
I never used debug and therefore this use case is not implemented yet.
I will look at it shortly.
Thank you

**Comment by davidnussio** [comment]
I just released the new version (4.2.0) with the requested functionality.
Thank you

![ksnip_20210623-175904](https://user-images.githubusercontent.com/324371/123134087-b7bcad80-d450-11eb-9e95-535d8f808e49.png)
